### ISSUE TITLE
Drop Sdl from struct names

### ISSUE THREAD

**Issue by adlesh** [nsauzede/vsdl2#3]
Could we think about dropping Sdl from the struct names? It seems a little redundant as the module's name will serve as reminder and prevent namespace conflicts. It would match the pattern of dropping the SDL_ prefix when translating to v. I'd be glad to do it myself; just wanted your input. 

```
struct SdlRect {

}
... becomes
struct Rect {

}

and vsdl2.SdlRect{} becomes vsdl2.Rect{}
```

**Comment by nsauzede** [comment]
absolutely !

**Comment by adlesh** [comment]
ok i'll get it w/ my image work. 
![image](https://user-images.githubusercontent.com/36427090/68534155-10cce180-02ff-11ea-9600-271ab94eb67d.png)

**Comment by nsauzede** [comment]
hey ! pretty nice the V logo ;-)

**Comment by adlesh** [comment]
closed by commit edfcceb
